### DISCUSSION TITLE
Custom scrollbars ✨

### DISCUSSION THREAD

**Discussion by luxonauta** [dracula/dracula-ui#62] [Ideas]
It would be nice if there were custom scrollbars, using the Dracula UI colour system (base and accent colours), with variables to customize the colours during the states manipulable by CSS.

And that was not applied by default but activated when adding classes to the DOM's ``<body>`` or other components.

🖖🏻🧑🏻

**Comment by zenorocha** [comment]
Ohhh I love that idea @luxonauta! 

A couple of days ago I was testing Dracula UI on Windows and I noticed how the scrollbars looked very ugly since they are typically optimized for light UIs and not dark UIs.

We should definitely have something like:

```html
<body class="drac-scrollbar">
```

Let me know if you want to send a PR with that.

**Reply by luxonauta** [reply]
Yes, sure! Soon I will send a PR with this idea! 😃
